packages/plugins/src/plugins/eigenlayer/jobs/withdrawalsJob.ts

```diff
@@ -0,0 +1,34 @@
+import { NetworkId } from '@sonarwatch/portfolio-core';
+
+import { Cache } from '../../../Cache';
+import { Job, JobExecutor } from '../../../Job';
+import { platformId } from '../constants';
+import { getEigenLayerWithdrawals } from '../helper';
+
+const executor: JobExecutor = async (cache: Cache) => {
+  // Get the YIELD positions
```

Comment:
`#fake-news` comment ? 

```suggestion
  // Get the withdrawal positions
```